apps/academy-wep/package.json

```
@@ -16,6 +16,7 @@
 		"@radix-ui/react-slot": "^1.1.2",
 		"@radix-ui/react-tooltip": "^1.1.8",
 		"autoprefixer": "^10.4.21",
```

Biome (1.9.4)

[error] The key autoprefixer was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)